starter-code/src/viking.js

```diff
@@ -1,11 +1,61 @@
 // Soldier
-class Soldier {}
+class Soldier {
+constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+}
+attack() {
+    return this.strength 
+}
+
+receiveDamage(damage) {
+    this.health -= damage
+}
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier{
+    constructor(name, health, strength){
+        super(health, strength)
+        this.name = name;
+    }
+    attack() {
+        return this.strength
+    }
+receiveDamage(damage) {
+    this.health -= damage
```

Comment:
because it'a an extend class of soldier and receiveDamage is already define in Solider, you could write: 
receiveDamage(damage){
  super.receiveDamage(damage)
  ... add your condition and sentences
}